templates/file-index.html

```diff
@@ -27,7 +27,7 @@ <h1 class="title">Files</h1>
       {{ range .Files }}
       <tr>
         <td class="is-vcentered" test-data-id="filename"><a href="/!{{ .ID }}">{{ .Filename }}</a></td>
-        <td class="is-vcentered">{{ .Size }}</td>
+        <td class="is-vcentered" pico-file-size>{{ formatFileSize .Size }}</td>
```

Comment:
We don't need the `pico-file-size` attribute anymore, right?

Reply:
Right. Can be removed :)